app/views/upload/index.html.erb

```diff
@@ -1,10 +1,11 @@
-<h1>Upload#index</h1>
-<p>Find me in app/views/upload/index.html.erb</p>
-<h1>Upload CSV File</h1>
+<h1 class="page-title">Upload</h1>
+<p>1. Kindly select a .csv file from your system to upload and visualize graduate outcome data.</p>
```

Comment:
Can we get rid of "and visualize" in this comment, it should be clear that this step will only upload a file and not visualize anything yet.